jekyll/_cci2/ios-tutorial.md

```diff
@@ -0,0 +1,154 @@
+
+For iOS projects, CircleCI generates a command to build and test your project using the `xcodebuild` command line tool. The generated command is similar to:
+
+```
+set -o pipefail &&
+  xcodebuild
+    CODE_SIGNING_REQUIRED=NO
+    CODE_SIGN_IDENTITY=
+    PROVISIONING_PROFILE=
+    -sdk iphonesimulator
+    -destination 'platform=iOS Simulator,OS=9.0,name=iPhone 6'
+    -workspace MyWorkspace.xcworkspace
+    -scheme "My Scheme"
+    clean build test |
+      tee $CIRCLE_ARTIFACTS/xcode_raw.log |
+      xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
+```
+
+If your project uses React Native, CircleCI automatically runs the `test` script phase from your `package.json`.
+
+## Code Signing
+
+It is possible for CircleCI to automatically inject your code signing certificates and unlock the keychain for your build. To use this automated code signing support for your iOS app, perform the following steps:
+
+1. Export your certificates (p12) by opening the Keychain Access.app, and selecting My Certificates in the left menu.
+![Keychain Access.app with the keychain that contains the keys, and My Certificates selected](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-access-my-certificates.png)
+A certificate with **iPhone Developer:** or **iPhone Distribution:** appears.
+
+2. Click the certificate, then select File > Export Items from the macOS menu bar ensuring that the file format is Personal Information Exchange (.p12). 
+![The Keychain Access.app file menu with Export Items in a hover state](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-file-menu.png)
+If the option is not available, check that you selected the private key when you selected the certificate.
+![The Keychain Access.app Export dialogue](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-keychain-export-dialogue.png)
+
+3. It is best practice to enter a certificate export password when it is requested. 
+
+## Adding your certificate
+
+1. Go to your project page in the CircleCI app, and click the Settings icon.
+
+2. Click the Code Signing link in the Permissions section.
+![The CircleCI Project Settings, iOS Code Signing page](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-cci-code-signing-menu.png)
+
+3. Click the Upload Key button and enter the details for your certificate, including the password you used when exporting the .p12, the `.p12` file you want to upload and click the Upload button.
+![The CircleCI Certificate Details](  {{ site.baseurl }}/assets/img/docs/ios-getting-started-cci-certificate-details.png)
+
+The uploaded p12 certificates will be installed into `circle.keychain` as part of your build setup. The password for this keychain is `circle`, and it is unlocked for the duration of the build. This keychain is also added to the Xcode search path, so any credentials stored here will be available to Xcode.
+
+## Using A Provisioning Profile
```

Comment:
This no longer applies as using fastlane match (new recommended way we will describe in the code signing doc) will take care of this. Should delete this section

Reply:
Or better still, remove contents and link to https://codesigning.guide for now